apps/crn-server/test/controllers/news.test.ts

```diff
@@ -120,6 +120,164 @@ describe('News controller', () => {
 
       expect(result.items[0]!.thumbnail).toBeUndefined();
     });
+
+    test('Should query data properly when request does not have options', async () => {
+      nock(baseUrl)
+        .get(`/api/content/${appName}/news-and-events`)
+        .query({
+          q: JSON.stringify({
+            take: 8,
+            filter: { path: 'data.type.iv', op: 'ne', value: 'Tutorial' },
+            sort: [{ order: 'descending', path: 'created' }],
+          }),
+        })
+        .reply(200, newsSquidexApiResponse);
```

Comment:
Although this newsSquidexApiResponse does not have appropriate data for this test, my understanding was that these tests are checking the query not the functionality itself. That's why I didn't change the newsSquidexApiResponse throughout the tests. But I am not sure if that's the best thing to do. What do you all think? :)